gst/nnstreamer/elements/gsttensor_trainer.c

```diff
@@ -493,6 +551,15 @@ gst_tensor_trainer_chain (GstPad * sinkpad, GstObject * parent,
   GstTensorMemory out_tensors[NNS_TENSOR_SIZE_LIMIT];
   GstTensorMetaInfo in_meta[NNS_TENSOR_SIZE_LIMIT];
   GstTensorMetaInfo out_meta[NNS_TENSOR_SIZE_LIMIT];
+  enum
+  {
+    TRAINING_LOSS,
+    TRAINING_ACCURACY,
+    VALIDATION_LOSS,
+    VALIDATION_ACCURACY
+  };
+  double data[4] = { -INFINITY, -INFINITY, -INFINITY, -INFINITY };
```

Comment:
Please change the integer literal, 4, to a constant variable.

Reply:
okay, I will fix it.